### DISCUSSION TITLE
fast way to open speaker profile picture

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1485] [Ideas] [answered]
Hi,

often we have the case, that we want to quick look on the picture of the speaker to remember who it is.
At the moment, when you click on the file it opens a new tab and there downloads the picture. Than you have to go to the downloads folder an open it.

Is there a way to hover over the picture or that it will open it in an new tab?

the issue sounds very special and nonsensical, but if you handle more than 100 speaker, it is really anoying.. :)

Thanks a lot!

**Comment by rixx** [comment]
Where do you click on the file? If it's on the profile page of the speaker … isn't the picture displayed at the very top already? Why open it separately?

**Reply by navlisilvan** [reply]
Yes the picture is displayed but very small. For our Speaker we mostly need to download it to recognize them.

**Comment by rixx** [comment, marked as answer]
This was improved in pretalx in the meantime – pretty much all instances of speaker names include a (tiny) version of their profile picture, and clicking it will either open the speaker page (if the name is a link) or open a larger version of the image (if the name is not already a link).
